go.mod

```diff
@@ -1,29 +1,27 @@
 module github.com/openebs/node-disk-manager
 
-go 1.14
+go 1.15
```

Comment:
When updating go version here, also update the go version used for building the binary in the correspnding `build/<component>/Dockerfile`